### PR TITLE
WIP: Fix elf sht null

### PR DESCRIPTION
**Pull request by ret2libc**
Need to add tests.

### CODE REVIEW

File: libr/bin/bin.c [hunk 1]
```diff
@@ -168,6 +168,17 @@ R_API void r_bin_info_free(RBinInfo *rb) {
 	free (rb);
 }
 
+R_API RBinImport *r_bin_import_clone(RBinImport *o) {
+	RBinImport *res = r_mem_dup (o, sizeof (*o));
+	if (!res) {
+		return NULL;
+	}
+	res->name = o->name ? strdup (o->name) : NULL;
```
Comment (radare) [inline libr/bin/bin.c]: use r_str_dup, which already does this nullchk for you, or maybe better use a macro

Comment (ret2libc) [reply libr/bin/bin.c]: kk

**Comment by ret2libc** [comment]
FIXME: _init and _fini functions are not correctly identified anymore
FIXME: adjust tests because there are wrong symbols in many of them
FIXME: consider whether we want "sym._10" symbols, etc.

**Comment by radare** [comment]
Looks red

**Comment by radare** [comment]
why did you reverted this?

screen shot 2018-06-25 at 23 11 40

**Comment by radare** [comment]
looks good to merge

**Pull request merged**